src/components/Breadcrumbs/index.tsx

```diff
@@ -16,51 +18,58 @@ interface BreadcrumbsProps {
 const Breadcrumbs: React.VFC<BreadcrumbsProps> = ({ breadcrumbs }) => (
   <Container>
     {breadcrumbs.map((bc, index) => (
-      <Fragment
-        breadcrumb={bc}
-        key={`breadCrumb-${bc.text}`}
-        active={breadcrumbs.length - 1 === index}
-      />
+      <Fragment key={`breadCrumb-${bc.text}`}>
+        <BreadcrumbFragment breadcrumb={bc} />
+        {breadcrumbs.length - 1 !== index && (
+          <Icon data-cy="breadcrumb-chevron" glyph="ChevronRight" />
+        )}
+      </Fragment>
     ))}
   </Container>
 );
 
-interface FragmentProps {
+interface BreadcrumbFragmentProps {
   breadcrumb: Breadcrumb;
-  active: boolean;
 }
-const Fragment: React.VFC<FragmentProps> = ({ breadcrumb, active }) => {
+const BreadcrumbFragment: React.VFC<BreadcrumbFragmentProps> = ({
+  breadcrumb,
+}) => {
   const { text, to, onClick, ...rest } = breadcrumb;
-
-  return active || !to ? (
-    <H3 {...rest}>{text}</H3>
-  ) : (
-    <StyledP1 {...rest}>
-      <StyledRouterLink to={to} onClick={onClick}>
-        {text}
-      </StyledRouterLink>
-    </StyledP1>
+  const shouldTrimMessage = text?.length > 25;
+  const message = shouldTrimMessage ? trimStringFromMiddle(text, 25) : text;
+  return (
+    <ConditionalWrapper
+      condition={shouldTrimMessage}
```

Comment:
Does this work? `trimStringFromMiddle` should leave string unchanged if trimming it would have no effect

```suggestion
  const trimmedText = trimStringFromMiddle(text ?? "", 25);
  return (
    <ConditionalWrapper
      condition={trimmedText !== text}
```